mods/cnc/rules/defaults.yaml

```diff
@@ -223,7 +223,7 @@
 		Upgrades: hospitalheal
 		Prerequisites: hosp
 	DetectCloaked:
-		Range: 1c0
+		Range: 3c0
```

Comment:
What is the logic behind this change?  It makes stealth tanks much less stealthy.